uiowa/uiowa
# Feature: Section alignment updates in uids_base

This PR:

- Resolves https://github.com/uiowa/uiowa/issues/1457 by adding `flex-wrap: wrap` to all layout builder columns and updates the card default width to `flex: 1 1 100%`.  
- Adds a layout builder style for section alignment called **Alignment: Start** in order to shift the column alignment to the top of each section.  By default content is distributed to fill the entire section so that card rows all appear at equal heights by default. 

# How to test

`blt frontend`
`blt ds --site=uiowa.edu` 

## How to test stacking issue

Using a site with layout builder enabled, try adding block components to columns within the two, three, and four column layout options.   
- Verify that they visually stack within the layout builder editor preview and remain stacked when the page is saved and does not look like https://github.com/uiowa/uiowa/issues/1457.    
- Stack two cards on top of each other
- Stack a text area with embedded image, a CTA
- Stack any other combinations and verify that they stack on top of each other

## How to test new alignment style

The new **Alignment: Start** style is meant to override the default alignment of each section. 

Example section with default alignment using a text area block on top of a cta block:

<img width="1751" alt="Screen Shot 2020-07-09 at 10 26 47 AM" src="https://user-images.githubusercontent.com/1036433/87059372-d89fda80-c1ce-11ea-8aa3-9f9f32079e4e.png">

Same section with **Alignment: Start**  applied to it:

<img width="1530" alt="Screen Shot 2020-07-09 at 10 26 56 AM" src="https://user-images.githubusercontent.com/1036433/87059410-e48b9c80-c1ce-11ea-8523-511c0b69e780.png">
